cypress/support/commands.js

```diff
@@ -18,3 +6,31 @@ Cypress.Commands.add('closeModal', () => {
 Cypress.Commands.add('useBodyReference', () => {
     cy.get('body').as('body');
 });
+
+// Mock data from events servers
+Cypress.Commands.add('mockEventsApi', () => {
+    cy.fixture('liveEventData.json').as('liveEventData');
+    cy.fixture('eventData.json').as('eventData');
+    cy.server();
+    cy.route(
+        '**/api/event/all/1?sort=-node_type_attributes.event_start',
+        '@eventData'
+    ).as('getEvents');
+    cy.route('**/api/event?status=Live', '@liveEventData').as('getLiveEvents');
+});
+
+// To stub requests with Cypress, we must remove fetch from the browser so
+// we can fallback to XMLHttpRequests
+// https://github.com/cypress-io/cypress-example-recipes/tree/master/examples/stubbing-spying__window-fetch#readme
+
+Cypress.on('window:before:load', win => {
+    delete win.fetch;
+});
+
+Cypress.Commands.add('visitWithoutFetch', path => {
```

Comment:
Thanks for the context/resources for mocking api calls. I like these helper functions as a means of doing this! When looking at this command as well as the `window:before:load` block I'm wondering if both of them are necessary. 

Does line 26 run on every cypress test and if so when would the `visitWithoutFetch` command be useful?

Reply:
Great catch, yep these are redundant. I opted to keep the `visitWithoutFetch` so it is run less often. Thanks for that!

Reply:
No problem! I really like the `visitWithoutFetch` helper as it gives you more control as to whether or not you want to mock api calls 👍

Reply:
Thanks! Yeah maybe we can use it on University as well somewhere